source/src/ports/generic_networkhandler.h

```diff
@@ -28,8 +28,8 @@
 #include "socket_timer.h"
 
 /*The port to be used per default for I/O messages on UDP.*/
-extern const int kOpenerEipIoUdpPort;
-extern const int kOpenerEthernetPort;
+extern const unsigned short kOpenerEipIoUdpPort;
+extern const unsigned short kOpenerEthernetPort;
```

Comment:
unsigned short has no guarantee on its length, therefore I would prefer uint16_t, which should also be available in recent versions of VS and would also satisfy the POSIX interface